README.md

```diff
@@ -1,50 +1,93 @@
+ * Integration testing of Deconst components, before you commit to
+   master and :shipit:
+ * Previewing local content or control repository changes
 
-### Prerequisites
+## Prerequisites
+### OSX
 
- * Install a recent version of [Docker](https://docs.docker.com/installation/#installation) for your platform.
- 
-If you are on MacOS, stop here and go to [Getting Started](#getting-started).
- 
-If you are on Windows or Linux, continue:
- 
- * Install [docker-compose](https://docs.docker.com/compose/install/). In addition to the `curl` command they list, you can also install it from [homebrew](http://brew.sh/) or [pip](https://pypi.python.org/pypi/docker-compose/1.3.0rc1).
+ * Install a recent version of
+   [Docker](https://docs.docker.com/installation/#installation)
 
-Make sure that docker-machine is create and running first.
 
-```bash
-docker-machine create --driver virtualbox dev
-```
-
-Run `eval $(docker-machine env default)` in each shell you'll use to interact with Docker.
+### Windows or Linux
 
-### Getting Started
-
-Clone this repository and `cd` into the directory where you cloned the repo. Then:
-
-1. Customize your credentials and other settings. The `env`
-   settings require knowing your Rackspace Cloud account info
-   and also where you intend to clone the control repo, for example.
+ 1. Install [docker-compose](https://docs.docker.com/compose/install/)
+ 1. Create a `docker-machine dev` instance:
     ```bash
-    cp env.example env
-    ${EDITOR} env
+    docker-machine create --driver virtualbox dev
     ```
-1. Launch the services.
-   ```bash
-   script/up
-   ```
+ 1. Run `eval $(docker-machine env default)` in each shell that you'll
+    use to interact with Docker.
+
+
+## Edit the env file
+
+Clone this Deconst Integrated repository and change to the directory
+where you cloned the repo. Customize your credentials and other
+settings in the `env` file. Setting the variables correctly require
+knowing your Rackspace Cloud account info and also where you intend to
+clone the control repo.
+
+   1. Copy the example file to `env`:
+      ```bash
+      cp env.example env
+      ```
+   1. Edit the `env` file in a text editor and change it as
+      appropriate for your environment.
+   1. Since this a local test environment, any API key is fine. To
+      create a random API key, type:
+      ```bash
+      hexdump -v -e '1/1 "%.2x"' -n 40 /dev/random
+      50d1606d4bd6bd1a5f2adefdcae603deff9b012a164cb8bf0b68caf3638d8e868
+      ```
+
+   1. Paste the key between the quotes on this line in the
+      `env` file:
+      ```bash
+      # The content service's administrative API Key. Used for write actions.
+      # Set this to something arbitrary and "random". While you're running locally it doesn't need to
+      # be anything particularly difficult.
+      export ADMIN_APIKEY="50d1606d4bd6bd1a5f2adefdcae603deff9b012a164cb8bf0b68caf3638d8e868""
```

Comment:
Remove the extra `"` at the end of this line.

Reply:
Actually, remove the quotes altogether. None of the env files I work with needed them; did you all run into errors when you did it without the quotes?